schemas/dab.draft.schema.json

```diff
@@ -133,6 +133,10 @@
             "enabled": {
               "type": "boolean",
               "description": "Allow enabling/disabling REST requests for all entities."
+            },
+            "request-body-strict": {
+              "type": "boolean",
```

Comment:
Should we also include `"default": true`?

Reply:
Yeah, we should. Thanks!